QUEST_LV_0200.tsv

```diff
@@ -206,7 +206,7 @@ QUEST_LV_0200_20150317_000205	{memo X}If you are not in a hurry, please defeat t
 QUEST_LV_0200_20150317_000206	{memo X}Rebel Commander Ruklys, and the citizens here engaged in an evil conduct{nl}despite the selections from the great merchant, Premier Eminent. The result is this place.
 QUEST_LV_0200_20150317_000207	{memo X}That's not bad. Maybe the guards can use it.
 QUEST_LV_0200_20150317_000208	{memo X}After Medis Diena, the Stone Frosts are occuring irregularly.{nl}Due to irregular Stone Frosts, we lost many of our soldiers on this wasteland.{nl}And we set the petrification devices limitedly here at the square of lamentation and we check the frequency of the frosts and move out the soldiers accordingly.
-QUEST_LV_0200_20150317_000209	{memo X}Check the petrification devices if you want to bo loyal to the Kingdom and receive it's determination.
+QUEST_LV_0200_20150317_000209	{memo X}Check the petrification devices if you want to to loyal to the Kingdom and receive it's determination.
```

Comment:
and "it's" too? / I wanted change too "a leader of a group" to "a group leader" =)